scripts/crontab/crontab.tpl

```diff
@@ -31,6 +31,7 @@ HOME=/tmp
 # Once per day after metrics import is done
 30 12 * * * %(z_cron)s update_addon_weekly_downloads
 30 13 * * * %(z_cron)s update_addon_average_daily_users
+30 14 * * * %(z_cron)s add_high_adu_extensions_to_notable
```

Comment:
https://github.com/mozilla/addons-server/pull/19999/commits/189aa31064ec2a90bc9fbd406c25c471a4658994

I added a later commit https://github.com/mozilla/addons-server/pull/19999/commits/081a29acba3054fd9ba15811d4e9ea073e986153 that uses the ADU numbers from bigquery because I wasn't sure if the database would sync up in time.

Reply:
Two thoughts though:
- we don't need to wait for `update_addon_average_daily_users` to finish if we use the ADU numbers directly so could run the tasks in parallel? 
- is celery going to baulk at passing down the entire `counts` list?  We _could_ filter it to just the required ADU before the task is fired, but it'd be splitting the logic 😒

Reply:
If you decorate `add_high_adu_extensions_to_notable` with `@use_primary_db`, then together with the fact that it's a `chord` callback the database should be updated by the time the task is ran, so you don't need BigQuery and you can keep the regular database query you had before

Reply:
yeah?  I wasn't sure if this was a "better" solution or not... it makes the code more involved though.

Reply:
The less we depend on BigQuery the better, IMHO. And as you said, passing down the full `counts` is risky for celery, and filtering it before not ideal because it'd split the logic, so might as well just require the primary database and avoid all that...

Reply:
https://github.com/mozilla/addons-server/pull/19999/commits/24dcdca2ebbfbf04b80a14a3127e7cd266f43775